ably/CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+# CHANGELOG - Ably
+
+## 1.0.0 / 2023-01-03
```

Comment:
```suggestion
## 1.0.0 / 2023-03-15
```

Reply:
Updated in c896408ce9a719623bafcf5b06198c1e8bdc67ae